test_lottogame.py

```diff
@@ -0,0 +1,107 @@
+import unittest
+from unittest.mock import patch
```

Comment:
In this example could be better to have a single test file for each class to test and not all in a single file

So if I have 3 class in 3 different file you could create:
test_lotto.py > TestLotto
test_bill.py > TestBill
etc